actionpack/test/controller/log_subscriber_test.rb

```diff
@@ -98,7 +98,7 @@ def setup
 
     @old_logger = ActionController::Base.logger
 
-    @cache_path = File.join Dir.tmpdir, Dir::Tmpname.make_tmpname("tmp", "cache")
+    @cache_path = Tempfile.create(%w"tmp cache", Dir.tmpdir)
```

Comment:
Seems should be here:

```ruby
@cache_path = Tempfile.create(%w"tmp cache", Dir.tmpdir).path
FileUtils.rm(@cache_path)
```

Reply:
Thank you for the info.
Then it should use `Dir.mktmpdir`.